### ISSUE TITLE
makepkg fails on Arch Linux

### ISSUE THREAD

**Issue by alexandre-mbm** [diegorubin/gnomato#49]
```
In file included from main.cc:38:0:
win_main.h:32:54: fatal error: boost/date_time/posix_time/posix_time.hpp: Arquivo ou diretório não encontrado
 #include "boost/date_time/posix_time/posix_time.hpp" 
                                                      ^
compilation terminated.
Makefile:438: recipe for target 'gnomato-main.o' failed
make[1]: *** [gnomato-main.o] Error 1
make[1]: Leaving directory '/tmp/yaourt-tmp-alexandre/aur-gnomato/src/gnomato-1.0.2/src'
Makefile:444: recipe for target 'all-recursive' failed
make: *** [all-recursive] Error 1
```

**Comment by diegorubin** [comment]
You need to install the libboost-date-time.
I created an isseu to put this verification in configure(#50).

Maybe this verification should be in arch package also.

**Comment by alexandre-mbm** [comment]
``` console
$ pacman -Ss libboost-date-time  # none
$ yaourt -Ss libboost-date-time  # none
$ yaourt -Ss boost-date-time     # none
$ pacman -Ss boost-date-time     # none
$ pkgfile libboost-date-time     # none
$ pkgfile boost-date-time        # none
$ 
```

?

``` console
$ ls /usr/lib/ | grep boost | grep time
libboost_date_time.so
libboost_date_time.so.1.57.0
libboost_timer.so
libboost_timer.so.1.57.0
$ pkgfile libboost_date_time.so
extra/boost-libs
$ 
```

Then the package is the `boost-libs`, but it is installed.

What to do ?

**Comment by alexandre-mbm** [comment]
``` console
$ pkgfile posix_time.hpp
extra/boost
community/passenger
$ pkgfile -l boost | grep -i posix_time.hpp
extra/boost /usr/include/boost/date_time/posix_time/posix_time.hpp
$ pkgfile -l passenger | grep -i posix_time.hpp
community/passenger /usr/lib/passenger/ext/boost/date_time/posix_time/posix_time.hpp
$ pacman -Q boost
erro: pacote "boost" não foi encontrado
```

Then I installed the package `boost`.

**Success!** `makepkg` and install OK. With the packages `boost` and `boost-libs`, but... see #51.

**Comment by vinipsmaker** [comment]
Version 1.1.0 is failing to build on latest ArchLinux with a bunch of errors related to C++11:

```
g++ -DPACKAGE_NAME=\"Gnomato\" -DPACKAGE_TARNAME=\"gnomato\" -DPACKAGE_VERSION=\"1.0.2\" -DPACKAGE_STRING=\"Gnomato\ 1.0.2\" -DPACKAGE_BUGREPORT=\"rubin.diego@gmail.com\" -DPACKAGE_URL=\"\" -DPACKAGE=\"gnomato\" -DVERSION=\"1.0.2\" -DGETTEXT_PACKAGE=\"gnomato\" -DSTDC_HEADERS=1 -DHAVE_SYS_TYPES_H=1 -DHAVE_SYS_STAT_H=1 -DHAVE_STDLIB_H=1 -DHAVE_STRING_H=1 -DHAVE_MEMORY_H=1 -DHAVE_STRINGS_H=1 -DHAVE_INTTYPES_H=1 -DHAVE_STDINT_H=1 -DHAVE_UNISTD_H=1 -DHAVE_LOCALE_H=1 -DHAVE_LC_MESSAGES=1 -DHAVE_BIND_TEXTDOMAIN_CODESET=1 -DHAVE_GETTEXT=1 -DHAVE_DCGETTEXT=1 -DENABLE_NLS=1 -I.  -I/usr/include -DGNOMATO_DATADIR=\""/usr/share/gnomato"\" -pthread -I/usr/include/gtkmm-3.0 -I/usr/lib/gtkmm-3.0/include -I/usr/include/atkmm-1.6 -I/usr/include/gtk-3.0/unix-print -I/usr/include/gdkmm-3.0 -I/usr/lib/gdkmm-3.0/include -I/usr/include/giomm-2.4 -I/usr/lib/giomm-2.4/include -I/usr/include/pangomm-1.4 -I/usr/lib/pangomm-1.4/include -I/usr/include/glibmm-2.4 -I/usr/lib/glibmm-2.4/include -I/usr/include/gtk-3.0 -I/usr/include/at-spi2-atk/2.0 -I/usr/include/at-spi-2.0 -I/usr/include/dbus-1.0 -I/usr/lib/dbus-1.0/include -I/usr/include/gtk-3.0 -I/usr/include/gio-unix-2.0/ -I/usr/include/cairo -I/usr/include/pango-1.0 -I/usr/include/atk-1.0 -I/usr/include/cairo -I/usr/include/cairomm-1.0 -I/usr/lib/cairomm-1.0/include -I/usr/include/cairo -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/usr/include/libpng16 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/harfbuzz -I/usr/include/libdrm -I/usr/include/libpng16 -I/usr/include/sigc++-2.0 -I/usr/lib/sigc++-2.0/include -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -I/usr/include/python2.7 -DGNOMATO_LOCALEDIR=\"/usr/share/locale\" -D_FORTIFY_SOURCE=2  -march=x86-64 -mtune=generic -O2 -pipe -fstack-protector-strong -MT gnomato-dialog_preferences.o -MD -MP -MF .deps/gnomato-dialog_preferences.Tpo -c -o gnomato-dialog_preferences.o `test -f 'dialog_preferences.cc' || echo './'`dialog_preferences.cc
In file included from /usr/include/c++/5.2.0/type_traits:35:0,
                 from /usr/include/sigc++-2.0/sigc++/visit_each.h:22,
                 from /usr/include/sigc++-2.0/sigc++/functors/slot.h:6,
                 from /usr/include/sigc++-2.0/sigc++/signal_base.h:27,
                 from /usr/include/sigc++-2.0/sigc++/signal.h:8,
                 from /usr/include/sigc++-2.0/sigc++/sigc++.h:86,
                 from /usr/include/glibmm-2.4/glibmm/thread.h:51,
                 from /usr/include/glibmm-2.4/glibmm.h:87,
                 from /usr/include/gtkmm-3.0/gtkmm.h:87,
                 from main.cc:23:
/usr/include/c++/5.2.0/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
 #error This file requires compiler and library support for the \
  ^
In file included from /usr/include/c++/5.2.0/type_traits:35:0,
                 from /usr/include/sigc++-2.0/sigc++/visit_each.h:22,
                 from /usr/include/sigc++-2.0/sigc++/functors/slot.h:6,
                 from /usr/include/sigc++-2.0/sigc++/signal_base.h:27,
                 from /usr/include/sigc++-2.0/sigc++/signal.h:8,
                 from /usr/include/sigc++-2.0/sigc++/sigc++.h:86,
                 from /usr/include/glibmm-2.4/glibmm/thread.h:51,
                 from /usr/include/glibmm-2.4/glibmm.h:87,
                 from /usr/include/gtkmm-3.0/gtkmm.h:87,
                 from dialog_preferences.h:26,
                 from dialog_preferences.cc:23:
/usr/include/c++/5.2.0/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
 #error This file requires compiler and library support for the \
  ^
In file included from /usr/include/c++/5.2.0/type_traits:35:0,
                 from /usr/include/sigc++-2.0/sigc++/visit_each.h:22,
                 from /usr/include/sigc++-2.0/sigc++/functors/slot.h:6,
                 from /usr/include/sigc++-2.0/sigc++/signal_base.h:27,
                 from /usr/include/sigc++-2.0/sigc++/signal.h:8,
                 from /usr/include/sigc++-2.0/sigc++/sigc++.h:86,
                 from /usr/include/glibmm-2.4/glibmm/thread.h:51,
                 from /usr/include/glibmm-2.4/glibmm.h:87,
                 from /usr/include/gtkmm-3.0/gtkmm.h:87,
                 from win_main.h:26,
                 from win_main.cc:23:
/usr/include/c++/5.2.0/bits/c++0x_warning.h:32:2: error: #error This file requires compiler and library support for the ISO C++ 2011 standard. This support is currently experimental, and must be enabled with the -std=c++11 or -std=gnu++11 compiler options.
 #error This file requires compiler and library support for the \
  ^
^CMakefile:438: recipe for target 'gnomato-main.o' failed
make[1]: *** [gnomato-main.o] Interrupção
Makefile:452: recipe for target 'gnomato-win_main.o' failed
make[1]: *** [gnomato-win_main.o] Interrupção
Makefile:466: recipe for target 'gnomato-dialog_preferences.o' failed
make[1]: *** [gnomato-dialog_preferences.o] Interrupção
Makefile:444: recipe for target 'all-recursive' failed
make: *** [all-recursive] Interrupção
```
